ingestion/src/metadata/profiler/profiler/datalake_sampler.py

```diff
@@ -50,53 +47,28 @@ def __init__(
     def _fetch_rows(self, data_frame):
         from pandas import notnull  # pylint: disable=import-outside-toplevel
 
-        sampled_data_frame = data_frame.sample(
-            n=(int(self.profile_sample) or 100)
-            if self.profile_sample_type == ProfileSampleType.ROWS
-            else None,
-            frac=self.profile_sample
-            if self.profile_sample_type == ProfileSampleType.PERCENTAGE
-            else None,
-            random_state=random.randint(0, 100),
-            replace=True,
-        )
         return (
-            sampled_data_frame.astype(object)
+            data_frame.astype(object)
```

Comment:
why do we convert it to `object` here?

Reply:
we were modifying it to object as it was a basic fallback to Nan, modified it.